src/applications/vaos/services/appointment/transformers.js

```diff
@@ -73,7 +73,10 @@ function isVideoVisit(appt) {
  * @returns {String} Status
  */
 function getVistaStatus(appointment) {
-  if (getAppointmentType(appointment) === APPOINTMENT_TYPES.vaAppointment) {
+  if (
+    appointment.vdsAppointments?.length ||
```

Comment:
Rather than check the type, this lets us pull a status if we have either of the lists that would have it.